### PR TITLE
Try to add tc-stab setting

### PR DESCRIPTION
**Pull request by yutayu**
According to this url.
http://linuxhowtos.org/manpages/8/tc-stab.htm

### CODE REVIEW

**Comment by yutayu** [comment]
What I can do :)
but , if stab setting is nothing, I couldn't erase stab setting in root qdisc.

**Comment by yutayu** [comment]
This script doen't work properly yet .

**Comment by yutayu** [comment]
I want to get your suggestion.

**Comment by yutayu** [comment]
This tc-stab's melit adds certain ATM caliculation to hfsc qdisc.
We can get more strict and more efficient traffic shaping.

**Comment by robinsmidsrod** [comment]
I'm sorry, but it's quite hard to understand your English. Can you please try to explain in plain English which problem you're trying to solve with this pull request? I would also recommend that you modify the pull request to include reasonable defaults. The script should work by default, but just specifying the default values (interface name, up/down speed).

**Comment by yutayu** [comment]
I mean my edited script won't work , yet.

**Comment by yutayu** [comment]
Sorry for my broken english.
see below
http://wiki.linuxwall.info/doku.php/en%3Aressources%3Adossiers%3Anetworking%3Atraffic_control#dsl_and_atm_the_ox_that_believed_to_be_frog

Linklayer atm and overhead parameter is important especially dsl line.
Data is divided 48bytes packet and 5 bytes padding data per each atm packets on dsl line.

Overhead is pppoa,pppoe overhead per each IP packets.
If you have 10mbps dsl line , you cannot 10mbps full , because of 5 bytes atm padding  per each atm packets and overhead per each IP packet.

At least , we should think about this for traffic shaping.

**Comment by yutayu** [comment]
tc stab setting is for help qdisc calc.
For exmaple, in PPPoA line,
linklayer = atm
overhead = -4( = PPPoA overhead 10 - ethernet overhead 14 )

This setting helps certain qdisc calc.

**Comment by yutayu** [comment]
I gave up editting this scripts.
I couldn't write scripts soon :(
Very sorry.
